docs/reference/commandline/daemon.md

```diff
@@ -54,6 +54,7 @@ weight = -1
       --mtu=0                                Set the containers network MTU
       --disable-legacy-registry              Do not contact legacy registries
       -p, --pidfile="/var/run/docker.pid"    Path to use for daemon PID file
+      --raw-logs                             Output logs with timestamps and no coloring
```

Comment:
was trying to write a shorter version, will retool according to your preferences when i get a sec